src/main/java/bio/terra/cli/command/helperclasses/options/CreateResource.java

```diff
@@ -0,0 +1,21 @@
+package bio.terra.cli.command.helperclasses.options;
+
+import bio.terra.workspace.model.CloningInstructionsEnum;
+import picocli.CommandLine;
+
+/**
+ * Command helper class that defines the relevant options for create a new controlled or referenced
+ * Terra resource: {@link ResourceName} and --description, --cloning.
+ *
+ * <p>This class is meant to be used as a @CommandLine.Mixin.
+ */
+public class CreateResource extends ResourceName {
```

Comment:
I am very not sure about this being the right structure for the `command.helperclasses.options` classes. But I would like a few more examples of option sharing before making a different decision. I think this will next come up with the `terra resources update` commands, which will probably want to reuse some of the `terra resources create` options. So, expecting some reorganization in this area in that PR, but don't want to do it prematurely.

RE mixin classes using mixins: I did this originally, but changed it to use inheritance because of the chaining required to access values (e.g. `createControlledResourceOptions.createResourceOptions.name` instead of `createControlledResourceOptions.name`). If we start having weird inheritance patterns, then certainly chaining accessors is better, but I figured we're not there yet.

RE only using one option per mixin: This would make each accessor more reusable across different types of commands. I do think the common groupings (e.g. options that all `terra resources create ...` commands need) are helpful and want to preserve that somehow.